src/models/user.js

```diff
@@ -1,69 +1,52 @@
+import mongoose from 'mongoose';
 import bcrypt from 'bcrypt';
+import isEmail from 'validator/lib/isEmail';
+
+const UserSchema = new mongoose.Schema({
+  username: {
+    type: String,
+    unique: true,
+    required: true
+  },
+  email: {
+    type: String,
+    unique: true,
+    required: true,
+    validate: [isEmail, 'invalid email']
+  },
+  password: {
+    type:String,
```

Comment:
Use .prettierrc which is in the project for automatic formatting.

Reply:
using prettier with .prettierrc config